mlir/lib/IR/AttributeDetail.h

```diff
@@ -23,7 +23,9 @@
 #include "mlir/Support/ThreadLocalCache.h"
 #include "llvm/ADT/APFloat.h"
 #include "llvm/ADT/PointerIntPair.h"
+#include "llvm/Support/ErrorHandling.h"
```

Comment:
Why do you need error handling?

Reply:
This is left over from a previous commit, apologies. I would like to clarify though, since clangd tells me there are other headers unused here, shall I remove those too as an en-passant tidy? Notably,
- `"mlir/Support/StorageUniquer.h"`
- `"llvm/ADT/PointerIntPair.h"`
- `"llvm/Support/TrailingObjects.h"`